h/api/views.py

```diff
@@ -222,6 +224,9 @@ def _create_annotation(fields, user):
     annotation['user'] = user.id
     annotation['consumer'] = user.consumer.key
 
+    if nipsa.is_nipsad(user.id):
+        annotation["not_in_public_site_areas"] = True
```

Comment:
This was just to give users looking at the documents from our api a clue what it means (though I now recall that we discussed hiding it from the output..)